epub33/rs/index.html

```diff
@@ -551,28 +551,31 @@ <h3>Manifest</h3>
 				<h3>Spine</h3>
 
 				<p id="confreq-rs-spine-order" data-tests="#pkg-spine-order">Reading Systems MUST provide a means of
-					rendering the EPUB Publication in the order defined in the <code>spine</code>, which includes: 1)
-					recognizing the first primary <code>itemref</code> as the beginning of the default reading order;
-					and, 2) rendering successive primary items in the order given in the <code>spine</code>.</p>
+					rendering the EPUB Publication in the order defined in the <code>spine</code>, which includes:</p>
+				<ul>
+					<li>recognizing the first primary (linear) <code>itemref</code> as the beginning of the default
+						reading order; and,</li>
+					<li>rendering successive primary items in the order given in the <code>spine</code>.</li>
+				</ul>
 
 				<p id="confreq-rendition-rs-spine-nonlinear">When a user traverses the default reading order defined in
 					the <a data-cite="epub-33#sec-spine-elem">spine</a> [[EPUB-33]], a Reading System MAY automatically
 					skip <code>itemref</code> elements marked as non-linear. <span
 						id="confreq-rs-spine-nonlinear-activation" data-tests="#pkg-spine-nonlinear-activation">When a
 						user activates a hyperlink to a non-linear resource, however, Reading Systems MUST render the
 						referenced resource or a designated fallback.</span> Reading Systems MAY also provide the option
-					for users to select whether to skip non-linear content by default or not.</p>
+					for users to skip non-linear content by default or not.</p>
 
 				<p id="confreq-rs-spine-progression-default" data-tests="#pkg-spine-progression-default">If the EPUB
 					Creator has not specified the <code>page-progression-direction</code>, the Reading System MUST
-					assume the value of <code>default</code>. When the EPUB Creator has set the value of the
-						<code>page-progression-direction</code> to <code>default</code>, either explicitly or
-					implicitly, the Reading System can choose rendering direction.</p>
+					assume the value of <code>default</code>. When <code>page-progression-direction</code> is
+					<code>default</code>, either explicitly or implicitly, the Reading System can choose the rendering
```

Comment:
We probably don't need "either explicitly or implicitly" here.
```suggestion
					<code>default</code>, the Reading System can choose the rendering
```